docs/application/dotnet/guides/app-management/cion.md

```diff
@@ -0,0 +1,434 @@
+            public ConnectionResultEventArgs(ConnectionResult result)
+            {
+                Result = result;
+            }
+            public ConnectionResult Result { get; }
+        }
+
+        public class ConnectionRequestEventArgs : EventArgs
+        {
+            public ConnectionRequestEventArgs(PeerInfo peerInfo)
+            {
+                PeerInfo = peerInfo;
+            }
+            public PeerInfo PeerInfo { get; }
+        }
+
+        public class DataReceivedEventArgs : EventArgs
+        {
+            public DataReceivedEventArgs(byte[] data, PeerInfo peerInfo)
+            {
+                Data = data;
+                PeerInfo = peerInfo;
+            }
+            public byte[] Data { get; }
+            public PeerInfo PeerInfo { get; }
+            public byte[] ReplyData { get; set; }
+        }
+
+        public class DisconnectedEventArgs : EventArgs
+        {
+            public DisconnectedEventArgs(PeerInfo peerInfo)
+            {
+                PeerInfo = peerInfo;
+            }
+            public PeerInfo PeerInfo { get; }
+        }
+
+        public class PayloadReceivedEventArgs : EventArgs
+        {
+            public PayloadReceivedEventArgs(Payload payload, PeerInfo peerInfo, PayloadTransferStatus status)
+            {
+                Payload = payload;
+                PeerInfo = peerInfo;
+                Status = status;
+            }
+            public Payload Payload { get; }
+            public PeerInfo PeerInfo { get; }
+            public PayloadTransferStatus Status { get; }
+        }
+    }
+    ```
+    ```csharp
+    string serviceName = "TestService";
+    string displayName = "TestDisplayName";
+    var server = new CionServer(serviceName, displayName);
+    ```
+
+2. Server is ready to listen the requests from the client:
+
+    ```csharp
+    server.Listen();
+    ```
+
+3. Accept the connection:
+
+    ```csharp
+    server.ConnectionRequestEvent += (sender, e) =>
+    {
+        ClionServer.ConnectionRequestEventArgs eventArgs = e as CionServer.ConnectionRequestEventArgs;
+        server.Accept(eventArgs.PeerInfo);
+    };
+    ```
+
+4. Receive data and payload:
+
+    ```csharp
+    server.DataReceivedEvent += (sender, e) =>
+    {
+        CionServer.DataReceivedEventArgs eventArgs = e as CionServer.DataReceivedEventArgs;
+        // eventArgs.Data
+    };
+    ```
+    ```csharp
+    server.PayloadReceivedEvent += (sender, e) =>
+    {
+        CionServer.PayloadReceivedEventArgs eventArgs = e as CionServer.PayloadReceivedEventArgs;
+        // eventArgs.Payload
+    };
+    ```
+
+<a name="group"></a>
+## Communicate in group style
+
+Cion Group can be used to subscribe the topic to publish and receive payload with other group members.
+
+To communicate with other group members, use the [Tizen.Applications.Cion.GroupBase](https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.Cion.GroupBase.html) class as follows:
```

Comment:
Same as above comment (the symbolic link "latest" need to be fixed)

Reply:
link is changed to API9 now

Reply:
changed to latest